tools/isoplot/isoplot.xml

```diff
@@ -0,0 +1,155 @@
+<tool id='isoplot' name='Isoplot: Generate plots from isocor output' version='0.1.0'>
+    <requirements>
+        <requirement type='package' version='1.3.0'>isoplot</requirement>
+    </requirements>
+    <command><![CDATA[
+         isoplot '$datafile' 'galaxy' '$format' --value '$value' --template '$template_file'
+#if $SP.generate_barplot:
+    -bp
+#end if
+#if $SP.generate_meanplot:
+    -mb
+#end if
+#if $SP.generate_heatmap:
+    -hm
+#end if
+#if $SP.generate_clustermap:
+    -cm
+#end if
+#if $IP.generate_barplot:
+    -IB
+#end if
+#if $IP.generate_meanplot:
+    -IM
+#end if
+#if $IP.generate_heatmap:
+    -HM
+#end if
+#if $stack:
+    -s
+#end if
+#if $verbosity:
+    -v
+#end if
+#if $annotation:
+    -a
+#end if
+-g -z '$output2' > $output1 2> '$output3'
+    ]]></command>
+    <inputs>
+        <param name='datafile' type='data' format='csv, tabular' multiple='true' label='Upload data file' />
+        <param name='template_file' type='data' format='csv, xlsx' multiple='true' label='Template file' argument='--template'/>
+        <!--param name='run_name' type='text' label='Run name' /-->
+        <param name='format' type='select' label='Plot format'>
+            <option value='pdf'>pdf</option>
+            <option value='jpeg'>jpeg</option>
+            <option value='png'>png</option>
+            <option value='svg'>svg</option>
+            <option value='html'>html</option>
+        </param>
+        <param name='value' type='select' label='Data to plot' >
+            <option value='corrected_area'>Corrected Area</option>
+            <option value='isotopologue_fraction'>Isotopologue Fraction</option>
+            <option value='mean_enrichment'>Mean Enrichment</option>
+        </param>
+        <section name='SP' title='Select static plots to generate'>
+            <param type='boolean' name='generate_barplot' label='Flag to generate barplots' argument='-bp'

```

Comment:
you can use truevalue="-bp" and falsevalue="" here, it will also make your command section easier.